docusaurus/docs/Android/04-compose/05-message-components/03-message-list.mdx

```diff
@@ -157,6 +157,30 @@ MessageList(
 
 You can also provide your own instance of `LazyListState` based on your custom logic where you define the starting scroll position of the state, the offset and how it behaves based on user events. For example, when deep linking.
 
+## Previewing Attachments
+
+Out of the box previews are provided for the following attachment types: uploading, link, Giphy, image, video and file.
+
+### Image and Video
+
+Image and video attachments are previewed as thumbnails which can be displayed as a single tile or multiple ones depending on how many attachments are contained within the specific message. By default, these are rendered by `MediaAttachmentFactory`.
+
+:::note
+Attachment factories are used for previewing attachments. You can use those we provide out of the box, add to them, or replace them with your own. You can read more about them [here](../03-general-customization/01-chat-theme.mdx#streamattachmentfactories).
+:::
+
+In practice they appear as such:
+
+![Default Image and Video Attachment Previews](../../assets/compose_message_list_stock_media_attachments.png)
+
+Video thumbnails are a paid feature, with the pricing listed [here](https://getstream.io/chat/pricing/). They are enabled by default but can be turned off by setting the [ChatTheme](../03-general-customization/01-chat-theme.mdx) property `videoThumbnailsEnabled` to `false`.
```

Comment:
Again pricing. Maybe just add a tutorial on how to disable the thumbnails.

Reply:
Same as here: 

https://github.com/GetStream/stream-chat-android/pull/4373#discussion_r1029655108